docs/central-upgrade.rst

```diff
@@ -91,6 +92,33 @@ You'll be asked to confirm the removal of all dangling images. Agree by typing t
 Version-specific upgrade instructions
 --------------------------------------
 
+.. _central-upgrade-2024.2:
+
+Upgrading to Central v2024.2
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are no required special steps related to this upgrade. However, there are some optional changes that you may want to know about or opt into.
+
+1. Configuration no longer requires rebuilding
```

Comment:
Is there still some configuration outside of changes to `.env` that does require rebuilding? If so, it might be clearer for this to say "Changes to .env no longer require rebuilding".